src/Hl7.Fhir.Specification.Shared.Tests/Snapshot/SnapshotGeneratorManifestTests.cs

```diff
@@ -630,7 +630,7 @@ void VerifyRule(StructureDefinition output, EvaluationContext ctx, SnapshotGener
 
             var nav = output;
             var expr = _fhirPathCompiler.Compile(rule.FhirPath);
-            Assert.IsTrue(expr.Predicate(nav, ctx), $"FAILED Rule {i}: '{rule.Text}'");
+            Assert.IsTrue(expr.Predicate(nav.ToElementNode(), ctx), $"FAILED Rule {i}: '{rule.Text}'");
```

Comment:
We'll probably have to make overloads for Predicate() so you don't have to do this.